SportHub.API/appsettings.json

```diff
@@ -19,5 +19,10 @@
     "SenderEmail": "SportHubEmail@gmail.com",
     "SenderPassword": "jtoesxhgiveuzncr"
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "Azure": {
+    "BlobStorageConnectionString": "DefaultEndpointsProtocol=https;AccountName=blobstoragesporthub;AccountKey=90/WQZ8tVSUr+W99lBubbGb9RywCJTFcZovL7NbHZGl83VzzXy/P5IN8mXVORc4c6KM1l0dCiA9c+AStIY/Sig==;EndpointSuffix=core.windows.net",
```

Comment:
We need to test it online, this is unsecure to store string here

Reply:
as string to db, but we don't have a choice because on backend we doesn't have working .env. All configuration and secrets we store in appsettings